test/index.js

```diff
@@ -784,11 +791,12 @@ id :: a -> a
 
 The environment is empty! Polymorphic functions require a non-empty environment.
 `));
+
+    reset ();
```

Comment:
If this test fails, other tests will fail too. Maybe you can do: `test(...).finally(reset)`. *Applies to repeated diffs.*

It'll cause the reset to happen in the next tick, so you'd have to [run tests in sequence](https://github.com/bearror/oletus/#running-tests-in-sequence-instead-of-parallely). This can be as easy as adding `await` before every `test()`.

It might be a good idea to do that anyway, in case any of the tests will ever be async. Since you're doing tests that can affect each other's results (via mutation of a global config), just adding `await` is good practice, and signals intent if nothing else.